src/documents/management/commands/decrypt_documents.py

```diff
@@ -69,20 +69,20 @@ def __gpg_to_unencrypted(self, passphrase: str):
 
             document.storage_type = Document.STORAGE_TYPE_UNENCRYPTED
 
-            ext = os.path.splitext(document.filename)[1]
+            ext: str = Path(document.filename).suffix
 
             if not ext == ".gpg":
                 raise CommandError(
                     f"Abort: encrypted file {document.source_path} does not "
                     f"end with .gpg",
                 )
 
-            document.filename = os.path.splitext(document.filename)[0]
+            document.filename = Path(document.filename).stem
 
-            with open(document.source_path, "wb") as f:
+            with Path(document.source_path).open("wb") as f:
```

Comment:
Both source_path and thumbnail_path are already Paths.  This redundant conversion happens a few places